### DISCUSSION TITLE
When I create PR, the indicator under menu is not on Pull requests.

### DISCUSSION THREAD

**Discussion by psbss** [community/community#32267] [Pull Requests] [answered]
Hello developers 🙌.

I wonder why the indicator is not displayed on the pull request tab.

see this image ↓ 
![image](https://user-images.githubusercontent.com/24839924/189027380-72d5bdf9-65ce-419c-b34d-82f6297d1a8d.png)

I think the orange indicator is on the pull request tab.

**Comment by radar-blip** [comment]
You mean that orange line? It's working fine for me. 
So others can help, what browser are you using?

**Reply by psbss** [reply]
Yes that orange line.
When I submit a pull request, you will see a screen like the one in the image, but I think the orange line should be under the pull request tab.

I'm using Chrome v105.0.5195.102, not using AdBlock extension

**Reply by radar-blip** [reply]
You're right, the indicator should move to whichever tab you select. I double checked, its working fine on firefox.
Hopefully someone else using chrome will chime in. Probably just a small glitch.

**Comment by psbss** [comment, marked as answer]
close
